backend/src/main/java/ca/bc/gov/backendstartapi/entity/seedlot/Seedlot.java

```diff
@@ -0,0 +1,203 @@
+package ca.bc.gov.backendstartapi.entity.seedlot;
+
+import ca.bc.gov.backendstartapi.entity.embeddable.AuditInformation;
+import ca.bc.gov.backendstartapi.enums.GeneticClassEnum;
+import ca.bc.gov.backendstartapi.enums.SeedlotStatusEnum;
+import jakarta.persistence.Column;
+import jakarta.persistence.Embedded;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.io.Serial;
+import java.io.Serializable;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import lombok.Generated;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+/** A registered seedlot. */
+@Entity
+@Table(name = "seedlot")
+@NoArgsConstructor
+@RequiredArgsConstructor
+@Getter
+@Setter
+public class Seedlot implements Serializable {
+
+  @Generated @Serial private static final long serialVersionUID = -8731389710757531855L;
+
+  @Id
+  @Column(name = "seedlot_number", length = 5)
+  @NonNull
+  private String id;
+
+  @Column(name = "seedlot_status_code", length = 3, nullable = false)
+  @NonNull
+  private SeedlotStatusEnum status;
+
+  @Column(name = "seedlot_comment", length = 2000)
+  private String comment;
+
+  // region Applicant
+  @Column(name = "applicant_client_number", length = 8)
+  private String applicantClientNumber;
+
+  @Column(name = "applicant_locn_code", length = 2)
+  private String applicantLocationCode;
+
+  @Column(name = "applicant_email_address", length = 100)
+  private String applicantEmailAddress;
+  // endregion
+
+  // region Lot information
+  @Column(name = "vegetation_code", length = 8)
+  private String vegetationCode;
+
+  @Column(name = "genetic_class_code", length = 1)
+  private GeneticClassEnum geneticClassCode;
+
+  @Column(name = "seedlot_souce_code", length = 3)
+  private String seedlotSourceCode;
+
+  @Column(name = "to_be_registrd_ind")
+  private Boolean intendedForCrownLand;
```

Comment:
Would it, though? The column name is terrible, probably due to constraints of the DBMS used at the time of its creation, which, I believe, aren't even in our new DBMS anymore.

We still use the old name in our new data model because we believe it'll save us work with extraction, but do we gain something by not making this abstraction here?

Reply:
Probably yes, because when looking for errors, looking at a json response, or when dealing with this class instance, is very likely we try to search the property based on its name, and since is very different, it can make us confuse, forcing to look this class definition to make sure is the right property.